### DISCUSSION TITLE
有优雅地在事件回调中获取 event target 元素的办法吗？

### DISCUSSION THREAD

**Discussion by chihane** [ant-design/ant-design#31258] [Q&A]
我希望实现在 Select 选择后自动失焦的功能，但 onChange 回调只提供了 value 和 option 两个参数，没有事件本身。

我现在通过直接 getElementById 的方式解决了问题，但有优雅的办法实现这个需求吗？

我现在实现需求的代码：

```javascript
<Select
    id='indexSelect'
    onChange={(value) => {
        setSelectedCandidateCode(value);
        document.getElementById('indexSelect').blur();
    }}
>
```

我期待能实现的代码：

```javascript
<Select
    onChange={(event, value) => {
        setSelectedCandidateCode(value);
        event.target.blur();
    }}
>
```

**Comment by zendext** [comment]
用 ref 获取 select 实例，select 实例上有focus，blur方法
https://github.com/react-component/select/blob/master/src/Select.tsx#L85

**Reply by chihane** [reply]
好像和 getElementById 没有本质上的区别
